src/porepy/applications/md_grids/mdg_library.py

```diff
@@ -0,0 +1,84 @@
+"""Library of mixed-dimensional grids.
+for use in tests. Other usage should be covered by the model_geometries.
+
+"""
+
+from __future__ import annotations
+
+from typing import Literal, Tuple
+
+import porepy as pp
+
+from . import domains, fracture_sets
+
+
+def square_with_orthogonal_fractures(
+    grid_type: Literal["simplex", "cartesian", "tensor_grid"],
+    meshing_args: dict[str, float],
+    fracture_indices: list[int],
+    size: pp.number,
+    **meshing_kwargs,
+) -> Tuple[pp.MixedDimensionalGrid, pp.FractureNetwork2d]:
+    """Create a mixed-dimensional grid for a square domain with up to two
```

Comment:
Description should fit in only one line. A more detailed description can go below.

Reply:
I don't think that's a strict rule. At least, we haven't enforced it strictly in the past.